src/components/tools/tile-comments.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import { observer, inject } from "mobx-react";
+import { BaseComponent } from "../base";
+import { TileCommentsModelType, TileCommentModelType } from "../../models/tools/tile-comments";
+import { IToolApiMap } from "./tool-tile";
+
+import "./tile-comments.sass";
+
+interface IProps {
+  docKey: string;
+  model: TileCommentsModelType;
+  toolApiMap?: IToolApiMap;
+}
+
+@inject("stores")
+@observer
+export class TileCommentsComponent extends BaseComponent<IProps, {}> {
+
+  public render() {
+    const { user } = this.stores;
+    const { model } = this.props;
+    if (!model.comments.length) return null;
+    if (!model.visible) {
+      return this.renderClosed();
+    }
+    const { class: clazz } = this.stores;
+    return (
+      <div className="tile-comments open">
```

Comment:
I think I'm seeing the odd display state that we saw in the demo meeting last week.  Both the `tile-comments` div and the `comments` div below are getting scrollbars.  Was this the intention?

Reply:
Good point about the double scroll - I put up a patch to remove one of the scrollbars. Can't seem to reproduce the behavior on my machine though.

Anyway, also a good point that the UI might not be worth spending too much time on for now.